api/src/pcapi/core/offerers/validation.py

```diff
@@ -11,6 +14,19 @@
 VENUE_BANNER_MAX_SIZE = 200_000
 
 
+def check_existing_business_unit(business_unit_id: int, offerer=Offerer):
+    business_unit = BusinessUnit.query.get(business_unit_id)
+    if not business_unit:
+        errors = ApiErrors()
+        errors.add_error("businessUnitId", "Ce point de facturation n'existe pas.")
+        raise errors
```

Comment:
Proposition pour ici et en dessous:
```suggestion
        raise ApiErrors(errors={"businessUnitId": "Ce point de facturation n'existe pas."})
```